frontend/src/config.ts

```diff
@@ -56,12 +58,15 @@ export const REGEX_TELEGRAM =
 // Username
 export const MIN_USERNAME_LEN = 3;
 export const MAX_USERNAME_LEN = 30;
+
 /*
 Note: username can only consist of uppercase & lowercase alphabets and numbers, 
 and must be between 3 to 30 characters. Update this regex when length limitations change
 */
 export const REGEX_USERNAME = /^([A-Za-z0-9]){3,30}$/;
 
+export const MAX_UPLOAD_SIZE = 3145728;
```

Comment:
This is in bytes right?

Good to add prefix on what upload size this is for, and comments on what units.

For e.g. video size limit different from avatar size limit.

Reply:
agreee